CHANGELOG.md

```diff
@@ -23,6 +23,10 @@ to the original message using the error `ErrWorkspaceLockedCannotDelete` instead
 * Adds `ExpiredAt` field to `OrganizationToken`, `TeamToken`, and `UserToken`. This enhancement will be available in TFE release, v202305-1. @JuliannaTetreault [#672](https://github.com/hashicorp/go-tfe/pull/672)
 * Adds `ContextWithResponseHeaderHook` context for use with the ClientRequest Do method that allows callers to define a callback which receives raw http Response headers.  @apparentlymart [#689](https://github.com/hashicorp/go-tfe/pull/689)
 
+## Bug Fixes
```

Comment:
This should go under the `Unreleased` section since this change has not been released yet. Add the section to the top of the file if it does not exist:
```suggestion
# UNRELEASED

## Bug Fixes
```